doc/man7/provider-signature.pod

```diff
@@ -414,6 +414,15 @@ the error and set the approved "fips-indicator" to 0.
 This option is used by the OpenSSL FIPS provider, and breaks FIPS compliance if
 set to 0.
 
+=item "sign-check" (B<OSSL_SIGNATURE_PARAM_FIPS_SIGN_CHECK>) <int>
+
+If required this parameter should be set early via an init function.
+The default value of 1 causes an error when a signing algorithm is used. (This
+is triggered by deprecated signing algorithms).
+Setting this to 0 will ignore the error and set the approved "fips-indicator" to 0.
+This option is used by the OpenSSL FIPS provider, and breaks FIPS compliance if
+set to 0.
```

Comment:
I would suggest this formulation:
```suggestion
This option is used by the OpenSSL FIPS provider, and allows using FIPS
non-approved algorithm if set to 0.
```

Reply:
We've used the "breaks FIPS compliance" in quite a few places elsewhere.

Reply:
I think if we are going to do this it should be in a separate pass after the PR's are all in. (There are quite a few of them)